### DISCUSSION TITLE
xsd:dateTime issue

### DISCUSSION THREAD

**Discussion by joegregoryphd** [opencaesar/platform#24] [General]
**When I build my description project, the reasoner returns the following error:**

<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<testsuites name="http://siedepartment.com/DoD_Example/bundle">
  <testsuite name="Consistency">
    <testcase name="http://siedepartment.com/DoD_Example/bundle">
      <failure message="The literal value does not satisfy the datatype restriction"><![CDATA[
The literal value does not satisfy the datatype restriction

<?xml version="1.0"?>
<rdf:RDF xmlns="urn:unnamed:ontology#ont1033382559725501#"
     xml:base="urn:unnamed:ontology#ont1033382559725501"
     xmlns:owl="http://www.w3.org/2002/07/owl#"
     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
     xmlns:xml="http://www.w3.org/XML/1998/namespace"
     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
     xmlns:SIE_Foundation="http://siedepartment.com/SIE_Foundation/SIE_Foundation#">
    <owl:Ontology rdf:about="urn:unnamed:ontology#ont1033382559725501"/>
    

    

    

    

    <owl:DatatypeProperty rdf:about="http://siedepartment.com/SIE_Foundation/SIE_Foundation#hasBeginInstant">
        <rdfs:range rdf:resource="http://www.w3.org/2001/XMLSchema#dateTime"/>
    </owl:DatatypeProperty>
    

    

    

    

    <owl:NamedIndividual rdf:about="http://siedepartment.com/DoD_Example/Part1#IOTE1_TimePeriod">
        <SIE_Foundation:hasBeginInstant>2023-01-01T12:00:00</SIE_Foundation:hasBeginInstant>
    </owl:NamedIndividual>
</rdf:RDF>

]]></failure>
    </testcase>
  </testsuite>
</testsuites>

**I am defining my time periods as follows:**

ci OTE1_TimePeriod [
                hasBeginInstant “2023-01-01T12:00:0”
]

**How could I resolve this? Thanks**

**Comment by melaasar** [comment]
A quoted string without a ^^type is treated as a String literal leading to the inconsistency.

Change your dateTime literal to:
 
ci OTE1_TimePeriod [
                hasBeginInstant “2023-01-01T12:00:0”^^xsd:dateTime
]

**Comment by joegregoryphd** [comment]
Thanks, this resolved the issue.
